### PR TITLE
Support Tarantool 1.6+

### LINKED ISSUE #1: please update to support 1.6 proto
Hi

this is 1.5 protocol, could you please add support for 1.6?

Thanks!

### PR DESCRIPTION
**Pull request by ligurio**
Fixes #1

PR depends on PR #3

### CODE REVIEW

File: test.lua [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/usr/local/bin/tarantool
```
Comment (Totktonada) [inline test.lua:1]: I would prefer `#!/usr/bin/env tarantool`. For example, I have tarantool in `/usr/bin`, not `/usr/local/bin`, so current shebang would not work for me.

**Comment by Totktonada** [comment]
I think the time comes and now it worth to move:

* `tarantool.dis.lua` to `tarantool15.dissector.lua`
* `tarantool16.dissector.lua` to `tarantool.dissector.lua`

Or kinda. I think it would be less confusing for users, assuming that most users are on 1.10/2.*.

File: test.lua (hunk 1 above)
Comment (mikhainin) [reply test.lua:1]: Agree, will update once merged

**Comment by mikhainin** [comment]
@Totktonada, yeah indeed, it's been a while - let's actualise it

**Comment by Totktonada** [comment]
@negram Thanks for responses!

What do you think about moving this repository into https://github.com/tarantool organization? This way it'll be easier for a user to find it and easier for us to make updates.

Of course, you can continue maintaining it after the transition if you want; or let us support the repository if you're not interested anymore.

BTW, I would also add it to https://github.com/tarantool/awesome-tarantool/

**Pull request merged**